### PR TITLE
added users:confirm_all rake task

### PR DESCRIPTION
**Pull request by IlyaOsotov**
I added a `users:confirm_all` rake task (https://github.com/EBWiki/EBWiki/issues/3271 issue)

### CODE REVIEW

**Comment by IlyaOsotov** [comment]
Whoops, there are some troubles with CI but I don't know why. I see other PRs with the same problems. Is it ok @rlgreen91?

**Comment by trystant** [comment]
HI @IlyaOsotov , you've been added as a collaborator. On top of that, take a look at this [document](https://github.com/EBWiki/EBWiki/wiki/Common-EBWiki-startup-errors)

**Comment by IlyaOsotov** [comment]
> HI @IlyaOsotov , you've been added as a collaborator. On top of that, take a look at this [document](https://github.com/EBWiki/EBWiki/wiki/Common-EBWiki-startup-errors)

Hi @trystant. I can't find information about how to fix `ArgumentError: Missing required arguments: aws_access_key_id, aws_secret_access_key` in CI. I understand I need this for local dev, but how can I update CI. I build it again but this error executed again. You can see logs in Travis CI.

**Comment by rlgreen91** [comment]
Hi @IlyaOsotov, thanks for your contribution!  You don't have to worry about the CI - it will keep doing that so long as your PR comes from a separate fork.  I'll test everything out locally.

File: lib/tasks/users.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :users do
+  desc 'mark all users as confirmed'
+  task confirm_all: :environment do
+    p 'Start confirming users'
+    User.all.in_batches do |group|
+      group.update_all(confirmed_at: Time.now)
```
Comment (rlgreen91) [inline lib/tasks/users.rake]: Can you change this to use `Time.current` to accommodate timezone issues?

Comment (IlyaOsotov) [reply lib/tasks/users.rake]: Done

**Pull request merged**